ai21/version.py

```diff
@@ -1 +1 @@
-VERSION = "2.12.0"
```

Comment:
@asafgardin - should be fixed prior merge manually or it's auto generated?

Reply:
auto